app/models/webhook.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Webhook < ApplicationRecord
+  STATUS = %i[pending succeeded failed].freeze
```

Comment:
Not sure which scenario is covered with `pending`. On first it seems to me that we only need succeeded for 200 status code and `failed` for everything else. Probably I'm missing something:)?

Reply:
it can happen if we have some wait in the sidekiq's queue, it's the default status

Reply:
I see, I was thinking that object will be created inside a job

Reply:
in facts it is, but I'll prefer to keep the pending state, we never know!